### ISSUE TITLE
weather icon overlap day in the hourly curve

### ISSUE THREAD

**Issue by MattRGX** [Akylas/oss-weather#158]
I've noticed that randomly the day is overlap by the weather icon in the hourly graphic, here is an example :+1: 

![image](https://github.com/Akylas/oss-weather/assets/58740578/3ee44a51-d9d0-476b-8af7-459ed640f359)

**Comment by farfromrefug** [comment]
@MattRGX very strange. Never saw that. Will try to reproduce.
thanks

**Comment by MattRGX** [comment]
@farfromrefug. I've just open again the app, and i can't reproduce that bug. so i don't know what could have cause it...

**Comment by farfromrefug** [comment]
@MattRGX i have a hint it could be related to font scale: either through the app font scale setting, or with android settings.
Did you make any change there by any chance?

**Comment by MattRGX** [comment]
@farfromrefug yes i tried to change this setting but then I went back to the default size (1). So it is likely related to that setting I think.

**Comment by farfromrefug** [comment]
@MattRGX thanks i reproduced it. It happens in a edge case, but i could not find the root cause thus the fix yet :s

**Comment by trend-1** [comment]
I don't know if it is the same issue but I have an overlap between the hour and the day. Fontsize 1.0 (standard).

<img width="461" height="613" alt="Image" src="https://github.com/user-attachments/assets/704df3f2-81d6-4130-a026-4ddb4ac15daf" />

Setting the fontsize to 2.0 (max) "solves" that problem.  
I'm on v. 3.02.1 Build 178

**Comment by farfromrefug** [comment]
@trend-1 not the same issue but thank you for reporting. I thought I fixed it but as it is random I need to dig deeper. 
Will fix Thank you

**Comment by farfromrefug** [comment]
@trend-1 will be fixed in next version
